site/content/en/docs/Concepts/_index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Concepts"
+linkTitle: "Concepts"
+weight: 4
+description: >
+  Core Kueue Concepts
+---
+
+This section of the documentation helps you learn about the components, APIs and
```

Comment:
instead copying the files from /docs, I would remove those. We don't want to maintain two sets of files.

Reply:
I would make it as part of the same PR, so that the git history is transferred and we have 0 chance of divergent changes.